src/pages/DealPage/hooks.ts

```diff
@@ -42,7 +42,7 @@ const useLoadDealDeps = (dealId?: Deal["id"]) => {
 
     const owner = useQueryWithClient(
         [QueryKey.OWNERS, get(deal, ["data", "properties", "hubspot_owner_id"], 0)],
-        (client) =>  getOwnerService(client, get(deal, ["data", "properties", "hubspot_owner_id"], 0)),
+        (client) =>  getOwnerService(client, get(deal, ["data", "properties", "hubspot_owner_id"], "0")),
```

Comment:
No need for `lodash` native TS/JS has us covered with optional chaining.

Reply:
Here is the story for refactoring `lodash` https://app.shortcut.com/deskpro/story/90992/hubspot-improvements